From 7356f4429ce3c8c90eb37c8e1a7c404e3cf74a11 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 11 Oct 2024 16:04:31 +0800
Subject: [PATCH 10/12] AOSCOS: feat: default to <Control>equal for the zoom-in
 grab

This saves the user from an extra Shift combo.
---
 docs/man/kmscon.1.xml.in | 2 +-
 src/kmscon_conf.c        | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/docs/man/kmscon.1.xml.in b/docs/man/kmscon.1.xml.in
index 60b09aa..9d5295d 100644
--- a/docs/man/kmscon.1.xml.in
+++ b/docs/man/kmscon.1.xml.in
@@ -387,7 +387,7 @@
         <term><option>--grab-zoom-in {grab}</option></term>
         <listitem>
           <para>Increase font size of the current terminal.
-                (default: &lt;Ctrl&gt;plus)</para>
+                (default: &lt;Ctrl&gt;equal)</para>
         </listitem>
       </varlistentry>
 
diff --git a/src/kmscon_conf.c b/src/kmscon_conf.c
index 0dd0db3..212a2f7 100644
--- a/src/kmscon_conf.c
+++ b/src/kmscon_conf.c
@@ -120,7 +120,7 @@ static void print_help()
 		"\t                                  Shortcut to scroll page up\n"
 		"\t    --grab-page-down <grab>     [<Shift>Next]\n"
 		"\t                                  Shortcut to scroll page down\n"
-		"\t    --grab-zoom-in <grab>       [<Ctrl>plus]\n"
+		"\t    --grab-zoom-in <grab>       [<Ctrl>equal]\n"
 		"\t                                  Shortcut to increase font size\n"
 		"\t    --grab-zoom-out <grab>      [<Ctrl>minus]\n"
 		"\t                                  Shortcut to decrease font size\n"
@@ -627,7 +627,7 @@ static struct conf_grab def_grab_page_down =
 		CONF_SINGLE_GRAB(SHL_SHIFT_MASK, XKB_KEY_Next);
 
 static struct conf_grab def_grab_zoom_in =
-		CONF_SINGLE_GRAB(SHL_CONTROL_MASK, XKB_KEY_plus);
+		CONF_SINGLE_GRAB(SHL_CONTROL_MASK, XKB_KEY_equal);
 
 static struct conf_grab def_grab_zoom_out =
 		CONF_SINGLE_GRAB(SHL_CONTROL_MASK, XKB_KEY_minus);
-- 
2.47.0

